src/scores/processing/isoreg_impl.py

```diff
@@ -503,10 +503,13 @@ def _contiguous_mean_ir(
 ) -> np.ndarray:
     """
     Performs classical (i.e. for mean functional) contiguous quantile IR on tidied data x, y.
-    Uses sklearn implementation rather than supplying the mean solver function to `_contiguous_ir`,
-    as it is about 4 times faster (since it is optimised for mean).
+    Refactored to use scipy instead of scikit learn.
```

Comment:
Rather than saying that it was refactored to use scipy instead of scikit learn, perhaps it's worth saying that we use scipy instead of supplying the mean solver function to `_contiguous_ir` as it's much faster

Reply:
This looks to be included in the latest version.